src/Wpf.Ui.Gallery/Views/Pages/Navigation/BreadcrumbBarPage.xaml

```diff
@@ -26,10 +28,20 @@
             <system:String x:Key="PageCsharpUrl">https://github.com/lepoco/wpfui/blob/development/src/Wpf.Ui/Controls/BreadcrumbBar.cs</system:String>
 
             <Style BasedOn="{StaticResource {x:Type ui:BreadcrumbBarItem}}" TargetType="{x:Type ui:BreadcrumbBarItem}">
+                <Setter Property="Icon">
+                    <Setter.Value>
+                        <iconElements:IconSourceElement>
+                            <iconElements:IconSourceElement.IconSource>
```

Comment:
`IconSourceElement` should be in **xmlns:ui="http://schemas.lepo.co/wpfui/2022/xaml"** namespace